core/src/avm2/globals/flash/errors/EOFError.as

```diff
@@ -0,0 +1,21 @@
+// The initial version of this file was autogenerated from the official AS3 reference at
+// https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/errors/EOFError.html
+// by https://github.com/golfinq/ActionScript_Event_Builder
+// It won't be regenerated in the future, so feel free to edit and/or fix
+
+package flash.errors
+{
+
+    public dynamic class EOFError extends IOError
+    {
+        prototype.name = "EOFError"
+
+        public function EOFError(message:String = "", id:int = 0)
+        {
+            super(message, id);
+            this.name = prototype.name
```

Comment:
I don't think we're supposed to set the name in errors from the `flash.errors` package, see IllegalOperationError.as for more details.

Reply:
Thanks for pointing this out, it is now fixed and I wrote up a test for verification purposes